modules/basis/basis-datasource/src/main/java/com/sap/xsk/factory/DynamicDataSourceFactory.java

```diff
@@ -78,13 +81,14 @@ private int getTomcatVersion() {
 
   private void setTomcatObjectFactory(int tomcatVersion) throws IllegalAccessException, InstantiationException {
     Class<?> factoryClass;
+    int maxTomcatSupportedVersion = 7;
```

Comment:
Isn't it actually `tomcatVersion7` instead of `maxTomcatSupportedVersion`?

Reply:
To be honest I had problems naming this variable and I forgot I left it in this state. The idea here is that before tomcat 7 the class we are trying to load is placed in one classpath whereas in tomcat 8 and above it is placed in a different classpath. So I am not sure what an appropriate name would be